gcp-cloud-run/02-provider.tf

```diff
@@ -0,0 +1,10 @@
+
+#provider "google" {
+#  project = var.project
+#  region  = var.default_region
+#}
+#
+#provider "google-beta" {
+#  project = var.project
+#  region  = var.default_region
+#}
```

Comment:
Remove?

Reply:
No. This is there in case someone wants to use it as a self-contained project. Otherwise, as a module, this is not needed.